src/test/ERC1155.t.sol

```diff
@@ -1379,7 +1427,8 @@ contract ERC1155Test is DSTestPlus, ERC1155TokenReceiver {
 
         uint256 minLength = min3(ids.length, mintAmounts.length, transferAmounts.length);
 
-        if (minLength == 0) revert();
+        if (minLength == 0) return;
+        if (to == address(0)) to = address(0xDEAD);
```

Comment:
do we wanna use asssume for these cases instead?

Reply:
I wish `assume` was `require` instead instead of custom error syntax ;_;

Reply:
@brockelmore can I add this ^

Reply:
you mean `vm.require(condition)`?

Reply:
yes, think it makes more sense

Reply:
^^^ agree